Specs.md

```diff
@@ -43,7 +43,7 @@ Each packet has a checksum byte at the end, which is the negative sum of all oth
 First byte is destination address (0x82 = Itho mainboard).  
 Second byte is reply address (0x80 = dongle).  
 2 bytes: message code.  
-1 byte: message flags (it seems: `04` = request, `01` = reply).  
+1 byte: message flags (it seems:  `06` = write, `04` = request, `01` = reply).  
```

Comment:
When is "write" used? Do we have more details about it?

Reply:
I found out it is used to set a new value for a setting.

Set setting command:
`
82 80 24 10 06 (1b length = 13) (4b new setting value) 00 00 00 00 00 00 00 00 00 00 00 00 00 (1b setting index) 00 (1b checksum)
`

Current, min and max setting values can be requested with:
`
82 80 24 10 01 (1b length = 13) 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 (1b setting index) 00 (1b checksum)
`

Response format:
`
82 80 24 10 04 (1b length = 13) (4b current value) (4b min setting value) (4b max setting value) 00 00 00 00 (1b setting index) (1b data format) (1b checksum)
`